source/configure/authentication-configuration-settings.rst

```diff
@@ -4,6 +4,13 @@ Authentication configuration settings
 .. include:: ../_static/badges/allplans-cloud-selfhosted.rst
   :start-after: :nosearch:
 
+Mattermost supports up to four distinct, concurrent methods of **Authentication**:
+
+- An OpenID provider
+- A SAML provider
+- An LDAP instance (ex Active Directory, OpenLDAP)
```

Comment:
```suggestion
- An LDAP instance (e.g., Active Directory, OpenLDAP)
```

Reply:
Changed "ex" as example to "e.g.," to be consistent with existing documentation.

Reply:
Done @cwarnermm :)